lib/matplotlib/rcsetup.py

```diff
@@ -174,6 +174,26 @@ def validate_bool_maybe_none(b):
         raise ValueError('Could not convert "%s" to bool' % b)
 
 
+def _validate_date_converter(s):
+    s = validate_string(s)
+    try:
+        import matplotlib.dates as mdates
+        mdates._rcParam_helper.set_converter(s)
+    except Exception as e:
```

Comment:
add a comment that this can only fail at initial import time?

Reply:
Yeah - so this gives me a bit of pause.  Does this cause any import slowdowns?

Reply:
I guess you could write `mdates = sys.modules.get("matplotlib.dates"); if mdates: ...` as matplotlib.dates *will* be in sys.modules once everything is set up properly...

Reply:
Thanks, I prefer that....

Reply:
@jklymak do you still want to change how this is written?